plugins/3MFReader/ThreeMFReader.py

```diff
@@ -116,4 +120,11 @@ def read(self, file_name):
         except Exception as e:
             Logger.log("e", "exception occured in 3mf reader: %s", e)
 
+        try: # Selftest - There might be more functions that should fail 
+            result.getBoundingBox()
```

Comment:
Maybe you can expand this check to also see if the bounding box is not valid? Then you'd catch both the situation where we load an invalid file as well as an empty file.